File: 1835-find-xor-sum-of-all-pairs-bitwise-and/1835-find-xor-sum-of-all-pairs-bitwise-and.py
class Solution(object):
    def getXORSum(self, arr1, arr2):
        """
        :type arr1: List[int]
        :type arr2: List[int]
        :rtype: int
        """
        ora, orb = 0, 0
        for a in arr1: ora ^= a
        for b in arr2: orb ^= b
        return ora & orb

File: 1835-find-xor-sum-of-all-pairs-bitwise-and/1835-find-xor-sum-of-all-pairs-bitwise-and.java
class Solution {
    public int getXORSum(int[] arr1, int[] arr2) {
        int ora = 0, orb = 0;
        for(int a : arr1) ora ^= a;
        for(int b : arr2) orb ^= b;
        return ora & orb;
    }
}